packages/tradershub/src/features/cfd/screens/CFDCompareAccounts/CompareAccountsCard.tsx

```diff
@@ -0,0 +1,79 @@
+import React from 'react';
+import { Text } from '@deriv/quill-design';
+import { THooks, TPlatforms } from '../../../../types';
+import { CFDPlatforms } from '../../constants';
+import CompareAccountsButton from './CompareAccountsButton';
+import CompareAccountsDescription from './CompareAccountsDescription';
+import CompareAccountsPlatformLabel from './CompareAccountsPlatformLabel';
+import CompareAccountsTitleIcon from './CompareAccountsTitleIcon';
+import InstrumentsLabelHighlighted from './InstrumentsLabelHighlighted';
+
+type TCompareAccountsCard = {
+    isAccountAdded: boolean;
+    isDemo: boolean;
+    isEuRegion: boolean;
+    isEuUser: boolean;
+    marketType: THooks.AvailableMT5Accounts['market_type'];
+    platform: TPlatforms.All;
+    shortCode: THooks.AvailableMT5Accounts['shortcode'];
+};
+
+const CompareAccountsCard = ({
+    isAccountAdded,
+    isDemo,
+    isEuRegion,
+    isEuUser,
+    marketType,
+    platform,
+    shortCode,
+}: TCompareAccountsCard) => {
+    return (
+        <div>
+            <div className='w-[180px] lg:w-[270px] relative overflow-hidden rounded-1200 my-50 mx-500 border-solid border-[1px] border-system-light-hover-background bg-background-primary-base hover:shadow-230'>
+                <CompareAccountsPlatformLabel platform={platform} />
+                {platform === CFDPlatforms.CTRADER && (
+                    <div className='p-75 lg:p-[15px] absolute z-[1] flex items-center justify-center w-[150px] h-1000 bg-brand-orange text-system-light-primary-background translate-x-400 -translate-y-1000 rotate-45 lg:translate-x-[170px] transform'>
```

Comment:
you can use values such as `z-10` for this rather than hardcoding it

Reply:
I want to use z-index:1
z-10 is z-index:10

Reply:
Yes I am aware of that but the purpose of this is to layer it ontop of the whole app so the value doesn't matter, am I right?

Reply:
Kind of
Because others dont have values so it would be higher in this particular case